remove subscriptionName from Output configration

mido-app opened the pull request:
Function only needs to know subscription name when it reads message,
so "subscriptionName" is necessary for Trigger configration, but unnecessary for Output configration.

azurecla commented:
Hi __@charislove__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@charislove 
2 options: 
* Resolve this merge conflict
* Easier: Close this PR, and make the change again in another PR.

cjgronlund commented:
@charislove 
Thanks! :>)

charislove commented:
Closing this PR and opening another

mido-app committed:
remove subscriptionName from Output configration

Function only needs to know subscription name when it reads message,
so "subscriptionName" is necessary for Trigger configration, but unnecessary for Output configration.
